src/config/rpc.ts

```diff
@@ -1,29 +1,30 @@
 import { ChainId } from '@sushiswap/core-sdk'
 
 const rpc = {
-  [ChainId.ETHEREUM]: 'https://eth-mainnet.alchemyapi.io/v2/q1gSNoSMEzJms47Qn93f9-9Xg5clkmEC',
+  [ChainId.ETHEREUM]:
+    'https://api.sushirelay.com/v1' ?? 'https://ethmainnetalchemyapi.io/v2/q1gSNoSMEzJms47Qn93f9-9Xg5clkmEC',
```

Comment:
@sambacha why this URL are wrong?

`'https://ethmainnetalchemyapi.io/v2/q1gSNoSMEzJms47Qn93f9-9Xg5clkmEC'`

Reply:
yup